applications/RANSApplication/custom_utilities/rans_check_utilities.cpp

```diff
@@ -44,7 +45,9 @@ bool CheckIfModelPartExists(const Model& rModel, const std::string& rModelPartNa
 }
 
 template <typename TVariableType>
-bool CheckIfVariableExistsInModelPart(const ModelPart& rModelPart, const TVariableType& rVariable)
+bool CheckIfVariableExistsInModelPart(
+    const ModelPart& rModelPart,
+    const TVariableType& rVariable)
```

Comment:
Can't the `HasNodalSolutionStepVariable` from the `ModelPart` be used to remove this?

Reply:
done :)